blocks.js

```diff
@@ -7554,24 +7556,26 @@ InputSlotMorph.prototype.dropDownMenu = function (enableKeyboard) {
         if (Object.prototype.hasOwnProperty.call(choices, key)) {
             if (key[0] === '~') {
                 menu.addLine();
-        // } else if (key.indexOf('§_def') === 0) { 
-        // menu.addItem(choices[key].blockInstance(), choices[key]);
-            } else if (key.charCodeAt(key.length - 1) == 0x25ba) {
+         // } else if (key.indexOf('§_def') === 0) { 
+         //     menu.addItem(choices[key].blockInstance(), choices[key]);
+            } else if (key.charAt(key.length - 1) == '{') {
```

Comment:
Oh right.  It was all harder back when I was using untypable characters.  I'll clean that up.

As for the many JSHint complaints, as I said earlier, maybe 2% of them were mine; the rest are (presumably) Jens's code.